libvirt/resource_libvirt_domain_test.go

```diff
@@ -412,13 +412,18 @@ func TestAccLibvirtDomain_KernelInitrdCmdline(t *testing.T) {
 		name   = "terraform-test-domain"
 		kernel = "${libvirt_volume.kernel.id}"
 		initrd = "${libvirt_volume.initrd.id}"
-		cmdline {
-			foo = 1
-			bar = "bye"
-		}
-		cmdline {
-			foo = 2
+
+
+		cmdline = [ 
+			{
+			arg1 = "value1"
+			}
+		 ]
+		cmdline = [
+		 {
+			arg1 = "value2"
```

Comment:
@MalloZup did you run this testcase after the change? It fails here:

```
=== RUN   TestAccLibvirtDomain_KernelInitrdCmdline
--- FAIL: TestAccLibvirtDomain_KernelInitrdCmdline (0.00s)
    testing.go:568: Step 0 error: /tmp/tf-test083582606/main.tf:27,3-10: Attribute redefined; The argument "cmdline" was already set at /tmp/tf-test083582606/main.tf:22,3-10. Each argument may be set only once.
FAIL
```

Reply:
It's OK. Fix is coming as part of #600. Once that's merged, we can avoid breakage like this. :)

Reply:
@zeenix yep ! thx  I will submit a PR for this

Reply:
I think i forgot to push changes.. Anyways i'm testing the PR with testacc now locally.. THx for spotting it! :rocket:

Reply:
https://github.com/dmacvicar/terraform-provider-libvirt/pull/611/files